ynr/apps/candidates/management/commands/candidates_create_csv.py

```diff
@@ -66,65 +41,43 @@ def add_arguments(self, parser):
             help="Only output CSV for the election with this slug",
         )
 
-    def get_people(self, election, qs):
-        all_people = []
-        elected_people = []
-        for person in queryset_iterator(qs, self.complex_popolo_fields):
-            for d in person.as_list_of_dicts(
-                election,
-                base_url=self.options["site_base_url"],
-                redirects=self.redirects,
-            ):
-                all_people.append(d)
-                if d["elected"] == "True":
-                    elected_people.append(d)
-        return all_people, elected_people
+    def slug_to_file_name(self, slug):
+        return "{}-{}.csv".format(self.output_prefix, slug)
 
     def handle(self, **options):
         if options["election"]:
             try:
-                all_elections = [Election.objects.get(slug=options["election"])]
+                election = Election.objects.get(slug=options["election"])
+                election_slug = election.slug
             except Election.DoesNotExist:
                 message = "Couldn't find an election with slug {election_slug}"
                 raise CommandError(
                     message.format(election_slug=options["election"])
                 )
         else:
-            all_elections = list(Election.objects.all()) + [None]
```

Comment:
I think this change introduces an undesirable behaviour change where we've got elections with 0 candidates.

Before we would generate a CSV with 0 rows in that case, but now we don't generate a file. The list of CSV files still contains all elections though, so this means some links on the data page will be 404s.

Reply:
Good catch. I've pushed 3c3190e that deals with this. We will still have 404s in the gap between importing from EE and running the create CSV command, but that's no worse than we have right now (it's run every half an hour)